packages/frontend2/src/app/(new)/data-availability/summary/_components/table/columns.tsx

```diff
@@ -54,7 +110,10 @@ export const columns = [
   }),
   columnHelper.accessor('tvs', {
     header: 'Total value secured',
-    cell: (ctx) => `$${formatNumber(ctx.getValue(), 2)}`,
+    cell: (ctx) =>
+      ctx.row.original.usedIn.length > 0
+        ? `$${formatNumber(ctx.row.original.tvs, 2)}`
```

Comment:
i guess it should use `formatCurrency`

Reply:
nope, actually I used it and realised it formats 0 as <$0.01, which is not intended here

Reply:
as stupid as it sounds but it has option to disable that
```ts
formatCurrency(tvl, currency, {
        showLessThanMinimum: false,
      })
```